stackstorm-jinja/pack.yaml

```diff
@@ -0,0 +1,8 @@
+name: jinja
+description: Jinja template engine
+keywords:
+ - jinja
+ - template
+version: 0.1.0
+author: StackStorm dev
```

Comment:
"StackStorm, Inc."? Of course, there is no StackStorm, Inc., but that's what we used in the past, and most our author fields are filled in like that :)